src/Elastic.Apm/Api/Node.cs

```diff
@@ -0,0 +1,15 @@
+using Elastic.Apm.Helpers;
+using Elastic.Apm.Report.Serialization;
+using Newtonsoft.Json;
+
+namespace Elastic.Apm.Api
+{
+	public class Node
+	{
+		[JsonProperty("configured_name")]
+		[JsonConverter(typeof(TrimmedStringJsonConverter))]
+		public string ConfiguredName { get; set; }
+
+		public override string ToString() => new ToStringBuilder(nameof(Service)) { { nameof(ConfiguredName), ConfiguredName } }.ToString();
```

Comment:
It should be `nameof(Node)` instead of `nameof(Service)`, right?

Reply:
yeap, you're totally right. That's what it means to copy-paste methods

Reply:
Yeap, you're right. That's what it means to copy-paste methods. Fixed.